dspace-api/src/main/java/org/orcid/utils/DateUtils.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * =============================================================================
+ *
+ * ORCID (R) Open Source
+ * http://orcid.org
+ *
+ * Copyright (c) 2012-2014 ORCID, Inc.
+ * Licensed under an MIT-Style License (MIT)
+ * http://orcid.org/open-source-license
+ *
+ * This copyright and license information (including a link to the full license)
+ * shall be included in its entirety in all copies or substantial portion of
+ * the software.
+ *
+ * =============================================================================
+ */
```

Comment:
This license is triggering an error in our license-check task... did we expect this utils package to be included with the jaxb packaged jar, or is this a set of supplementary utils?

[ERROR] Failed to execute goal com.mycila:license-maven-plugin:2.11:check (check-headers) on project dspace-api: Some files do not have the expected license header -> [Help 1]

Reply:
Hmmm...looks like a few `org.orcid.utils.*` files accidentally snuck in here. Thanks for pointing these both out, @kshepherd! I don't believe either is need in our source code. But, I'll look into it and get it resolved.